recipe/meta.yaml

```diff
@@ -10,9 +10,10 @@ source:
   patches:
     # diff from source distribution to https://github.com/ampl/gsl/tree/v2.4.0
     - patch_cmake_for_windows.diff  # [win]
+    - xfails-on-ppc64le-for-tolerance.patch  # [not x86_64]
```

Comment:
Can you add the savannah link as a comment above?

Reply:
surely